### PR TITLE
[17.0][MIG] crm_phonecall_summary_predefined: Migration to 17.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
Standard migration to v17.

Depends on:

- [ crm_phonecall] https://github.com/OCA/crm/pull/540

@Tecnativa
TT49908
@victoralmau @pilarvargas-tecnativa

### CODE REVIEW

File: crm_phonecall_summary_predefined/readme/CONFIGURE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+To configure the possible summary options:
+
+1.  Activate the developer mode
+2.  Go to *CRM \> Configuration \> Leads & Opportunities \> Phone Calls
```
Comment (pilarvargas-tecnativa) [inline crm_phonecall_summary_predefined/readme/CONFIGURE.md]:
```suggestion
2.  Go to *CRM \> Configuration \> Phone Calls
```

**Review by pilarvargas-tecnativa — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]
Code review OK

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**